plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveWriteUtils.java

```diff
@@ -528,6 +528,16 @@ public static boolean isHdfsEncrypted(HdfsContext context, HdfsEnvironment hdfsE
         }
     }
 
+    public static boolean isFileCreatedByQuery(Path file, ConnectorSession connectorSession)
```

Comment:
This overload seems unused and anything this method does can be done by the other impl so I consider it redundant.